src/Controls/src/Core/Stepper/Stepper.Mapper.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace Microsoft.Maui.Controls;
+public partial class Stepper
+{
+    internal static new void RemapForControls()
+    {
+        StepperHandler.Mapper.AppendToMapping(nameof(Stepper.Increment), Stepper.MapInterval);
```

Comment:
Is there a reason this cannot be done in the Core project?

If I think the reason you need this is because Controls and Core have different names Interval vs Increment, you may just need to add this method:

```cs

		protected override void OnPropertyChanged([CallerMemberName] string? propertyName = null)
		{
			base.OnPropertyChanged(propertyName);

			if (propertyName == IncrementProperty.PropertyName)
			{
				Handler?.UpdateValue(nameof(IStepper.Interval));
			}
		}
```

This can be seen in some other controls with a mismatch of property names causing the automatic events to not flow.

Reply:
I think we have a few different ways to do this, but I am no longer sure the way we decided on:

* new mappers in Controls(this PR)
* overriding OnPropertyChanged (the comment)
* using BindableProperty.Create(propertyChanged: )